dsgrid/registry/dataset_registry_manager.py

```diff
@@ -361,8 +361,8 @@ def _update(
 
         return model
 
-    def remove(self, dataset_id: str):
-        config = self.get_by_id(dataset_id)
+    def remove(self, dataset_id: str, version: Optional[str] = None):
+        config = self.get_by_id(dataset_id, version=version)
```

Comment:
So `version is None` returns just the latest dataset with that id?

Reply:
Using the version for removals was a mistake. We always remove all versions. Removing by specific version would be a larger change. We might not want to ever do that. In most cases we want to remove everything and start over (for project or dataset).